core/src/main/java/org/apache/carbondata/core/util/CarbonProperties.java

```diff
@@ -983,8 +983,11 @@ public void print() {
     for (String levelSize : levels) {
       try {
         int size = Integer.parseInt(levelSize.trim());
-        if (validate(size, 100, 0, -1) < 0) {
+        if (validate(size, CarbonCommonConstants.COMPACTION_SEGMENT_LEVEL_MAX,
+            CarbonCommonConstants.COMPACTION_SEGMENT_LEVEL_MIN, -1) < 0) {
           // if given size is out of boundary then take default value for all levels.
+          LOGGER.warn("The COMPACTION_LEVEL_THRESHOLD is out of boundary, "
```

Comment:
print the default value also, similar log like line : 995

Reply:
solved